### ISSUE TITLE
Best way to handle a White Label with multiple schemes using different bundle_ids

### ISSUE THREAD

**Issue by kraftydevil** [fastlane/fastlane#787]
I'm trying to get started with fastlane but I'm having trouble based with my setup. Basically I have a white label app where I add a different target/scheme every time we get a new client. 

Using 'fastlane init' appears to assume my project uses only one bundle id. To the contrary, the apps generated by my project have different bundle ids based on the target/scheme.

In the command line, `gym --scheme aSchemeName --configuration aBuildConfigurationName` works great, but of course I want to be able to use a Fastfile to string together other actions associated with building.

The Fastfiles I've seen only reference one scheme per lane. Does that mean that the lane to bundle id relationship is 1-to-1?

In other words, can I build multiple apps in the same lane even if they each have different bundle ids? Or would I be better off creating a lane for each app/build configuration combination and then stringing those together somehow?

Any direction on how to handle a white label setup in a fastlane would be most helpful.

Imagined pseudo build process:
- For each app, create an ad hoc build and send it to Hockey.
- Optional: send to TestFlight for larger external testing.
- Optional: send to App Store along with the dSYM to Hockey as well.

**Comment by truth3** [comment]
@kraftydevil You can build multiple apps in the same lane with different bundle IDs (call gym multiple times). I have taken the approach of one lane per target and sometimes multiple lanes for the same target for publishing to TestFlight and also for App Store review.

You will want to create separate folders within the Deliver directory to maintain all of the assets separately, and can do so by creating a new directory and running `deliver init` Hope that helps!

**Comment by KrauseFx** [comment]
Please check out the [FAQs](https://github.com/fastlane/fastlane/blob/master/docs/FAQs.md#multiple-targets-of-the-same-underlying-app) and the [SunApps Example](https://github.com/fastlane/examples/tree/master/SunApps)

**Comment by kraftydevil** [comment]
.env files to the rescue! Thank you for the direction.

**Comment by superarts** [comment]
Link "https://github.com/fastlane/fastlane/blob/master/docs/FAQs.md#multiple-targets-of-the-same-underlying-app" is dead.
